src/index.ts

```diff
@@ -109,16 +127,16 @@ export interface UploadConfig {
    */
   origin?: string;
 
+  /**
+   * Specify query parameters that go along with the initial upload request. See
+   * https://cloud.google.com/storage/docs/json_api/v1/objects/insert#parameters
+   */
+  params?: QueryParameters;
+
   /**
    * Apply a predefined set of access controls to the created file.
    */
-  predefinedAcl?:
-    | 'authenticatedRead'
-    | 'bucketOwnerFullControl'
-    | 'bucketOwnerRead'
-    | 'private'
-    | 'projectPrivate'
-    | 'publicRead';
+  predefinedAcl?: QueryParameters['predefinedAcl'];
```

Comment:
For this, I'd export a PredefinedAcl type:

```js
export type PredefinedAcl = 
    | 'authenticatedRead'	
    | 'bucketOwnerFullControl'	
    | 'bucketOwnerRead'	
    | 'private'	
    | 'projectPrivate'	
    | 'publicRead';
```

Then down here, or other places where you need it, you can just use it as a type!

```js
predefinedAcl?: PredefinedAcl;
```